### PR TITLE
Drop TravisCI testing in favor of GitHub Actions, test up to PHP 8.0

### PR DESCRIPTION
**Pull request by mondrake**
TravisCI has become overly complicated to test on for OSS, most projects I follow shifted to GitHub Actions instead.

This PR removes TravisCI configuration, sets up GithHub actions for running on a weekly schedule for testing on all PHP versions from 5.4 to 8.0.

PHP 8.1 should be managed as a separate PR because it will require more changes (for example, #93).

### CODE REVIEW

File: tests/FontLib/FontTest.php [hunk 1]
```diff
@@ -7,12 +7,19 @@
...
+        // @todo when PHP 5.4 support is dropped, uncomment line below and drop
+        //       the try...catch block.
+        // $this->expectException('\Fontlib\Exception\FontNotFoundException');
+        try {
+            Font::load('non-existing/font.ttf');
+            $this->fail('Load should have failed.');
+        }
+        catch (\Fontlib\Exception\FontNotFoundException $e) {
+            // Avoid throwing a risky test error.
+            $this->assertTrue(true);
```
Comment (enumag) [inline tests/FontLib/FontTest.php:21]: You can use `expectNotToPerformAssertions()` instead.

Comment (mondrake) [reply tests/FontLib/FontTest.php:21]: Not sure in PHPUnit 4? Tests need to run on PHP 5.4 too.

Comment (enumag) [reply tests/FontLib/FontTest.php:21]: Although... not sure if it's available in old phpunit versions...

**Comment by mondrake** [comment]
ping @PhenX who seems the only person able to commit here

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: PHPUnit tests
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: "0 6 * * 3"
```
Comment (enumag) [inline .github/workflows/phpunit.yml:7]: This is unnecessary in my opinion. Since it doesn't even require any packages it can't really break on it's own.

Comment (mondrake) [reply .github/workflows/phpunit.yml]: Agree.

**Comment by bsweeney** [comment]
not really necessary for the next milestone, but I assigned there to remind me to take a look

**Review by bsweeney — APPROVED** [review]

**Pull request merged**